contracts/IPayoutable.sol

```diff
@@ -1,28 +1,45 @@
-    function putFundsInSuspenseInPayout(address orderer, string calldata operationId) external returns (bool);
-    function executePayout(address orderer, string calldata operationId) external returns (bool);
-    function rejectPayout(address orderer, string calldata operationId, string calldata reason) external returns (bool);
-    function isPayoutOperatorFor(address walletToDebit, address orderer) external view returns (bool);
-    function retrievePayoutData(address orderer, string calldata operationId) external view returns (address walletToDebit, uint256 value, string memory instructions, PayoutStatusCode status);
+    function processPayout(string calldata operationId) external returns (bool);
+    function putFundsInSuspenseInPayout(string calldata operationId) external returns (bool);
+    function executePayout(string calldata operationId) external returns (bool);
+    function rejectPayout(string calldata operationId, string calldata reason) external returns (bool);
+    function retrievePayoutData(string calldata operationId) external view returns (
+        address walletToDebit,
+        uint256 value,
+        string memory instructions,
+        PayoutStatusCode status
+    );
 
+    function authorizePayoutOperator(address operator) external returns (bool);
+    function revokePayoutOperator(address operator) external returns (bool);
+    function isPayoutOperatorFor(address operator, address from) external view returns (bool);
 
-    event PayoutOrdered(address indexed orderer, string indexed operationId, address indexed walletToDebit, uint256 value, string instructions);
-    event PayoutInProcess(address indexed orderer, string indexed operationId);
-    event PayoutFundsInSuspense(address indexed orderer, string indexed operationId);
-    event PayoutExecuted(address indexed orderer, string indexed operationId);
-    event PayoutRejected(address indexed orderer, string indexed operationId, string reason);
-    event PayoutCancelled(address indexed orderer, string indexed operationId);
-    event PayoutOperatorAuthorized(address indexed walletToBePaidOut, address indexed orderer);
-    event PayoutOperatorRevoked(address indexed walletToBePaidOut, address indexed orderer);
+    event PayoutOrdered(address indexed orderer, string operationId, address indexed walletToDebit, uint256 value, string instructions);
+    event PayoutInProcess(address indexed orderer, string operationId);
+    event PayoutFundsInSuspense(address indexed orderer, string operationId);
+    event PayoutExecuted(address indexed orderer, string operationId);
+    event PayoutRejected(address indexed orderer, string operationId, string reason);
+    event PayoutCancelled(address indexed orderer, string operationId);
+    event AuthorizedPayoutOperator(address indexed operator, address indexed account);
+    event RevokedPayoutOperator(address indexed operator, address indexed account);
```

Comment:
Why do we do this naming change when the original was good , in a paste sense as a normal event?